src/main/scala/amba/ahb/Bundles.scala

```diff
@@ -27,6 +27,27 @@ class AHBBundle(params: AHBBundleParameters) extends AHBBundleBase(params)
   val hreadyout = Bool(INPUT)
   val hresp     = Bool(INPUT)
   val hrdata    = UInt(INPUT, width = params.dataBits)
+
+  def tieoff() {
+    hreadyout.dir match {
+      case INPUT =>
+        hreadyout := Bool(false)
+        hresp     := Bool(false)
+        hrdata    := UInt(0)
+      case OUTPUT => 
+        hmastlock := Bool(false)
+        htrans    := UInt(0)
+        hsel      := Bool(false)
+        hready    := Bool(false)
+        hwrite    := Bool(false)
+        haddr     := UInt(0)
+        hsize     := UInt(0)
+        hburst    := UInt(0)
+        hprot     := UInt(0)
+        hwdata    := UInt(0)
+      case _ =>
+    }
+  }
 }
```

Comment:
Could we use the named constants here?
htrans := TRANS_IDLE
hburst := BURST_SINGLE
hprot := PROT_DEFAULT
hresp := RESP_OKAY

Reply:
I'll use the constants. I think the blocking-forever policy is the right one.